Quick_Deploy/vLLM/README.md

```diff
@@ -55,7 +55,7 @@ required to serve the [facebook/opt-125m](https://huggingface.co/facebook/opt-12
 ```
 mkdir -p model_repository/vllm_model/1
 wget -P model_repository/vllm_model/1 https://raw.githubusercontent.com/triton-inference-server/vllm_backend/main/samples/model_repository/vllm_model/1/model.json
-wget -P model_repository/vllm_model/ https://raw.githubusercontent.com/triton-inference-server/vllm_backend/main/samples/model_repository/vllm_model/config.pbtxt
+wget -P model_repository/vllm_model/ https://raw.githubusercontent.com/triton-inference-server/vllm_backend/912896b12d66ac63fc7c0758fc39317985e6bcd5/samples/model_repository/vllm_model/config.pbtxt
```

Comment:
```suggestion
wget -P model_repository/vllm_model/ https://raw.githubusercontent.com/triton-inference-server/vllm_backend/r<xx.yy>/samples/model_repository/vllm_model/config.pbtxt
```
where <xx.yy> is the version of Triton that you want to use. Please note, that Triton's vLLM container has been introduced starting from 23.10 release.

Reply:
How about we change it to something like the above? I believe the issue was with 23.10 container, since in 23.11 we've added autocomplete.

Reply:
Yes this makes sense!